driver/pace/driver/grid.py

```diff
@@ -196,6 +199,111 @@ def get_grid(
         return damping_coefficients, driver_grid_data, grid_data
 
 
+@GridInitializerSelector.register("external")
+@dataclasses.dataclass
+class ExternalNetcdfGridConfig(GridInitializer):
+    """
+    Configuration for grid initialized from external data.
+    Input is from tile files generated by FRE-NCtools methods
+    The ExternalNetcdfGridConfig get_grid member method calls
+    the MetricTerms class method from_generated which generates
+    an object of MetricTerms to be used to generate the
+    damping_coefficients, driver_grid_data, and grid_data variables
+    """
+
+    grid_type: Optional[int] = 0
+    grid_file_path: str = "/input/tilefile"
+
+    # TODO: Area read in?
```

Comment:
Log it ?

Reply:
The area is read-in I will remove this TODO.